### ISSUE TITLE
composer_version documentation should specify to nullify composer_version_branch

### ISSUE THREAD

**Issue by brandonsavage** [geerlingguy/ansible-role-composer#77]
Composer recently released version 2.2 which includes BC breaking changes. However, limiting the version with composer_version doesn't seem to do anything.

My specific configuration:

```yml
composer_version: '2.1.14'
```

What I get when I run `composer --version`:
```bash
root@ansible-testing-2022-brandon:~# composer --version
Do not run Composer as root/super user! See https://getcomposer.org/root for details
Continue as root/super user [yes]? yes
Composer version 2.2.3 2021-12-31 12:18:53
```

**Comment by brandonsavage** [comment]
It appears my mistake was not nullifying the `composer_verrsion_branch` parameter. The documentation should be more explicit. I'll see if I can make a PR but I filed this to keep track in case I forget.

**Comment by gdubicki** [comment]
Hi @brandonsavage! I was just hit by the same thing and I created https://github.com/geerlingguy/ansible-role-composer/pull/79 to prevent this in the future.

**Comment by gdubicki** [comment]
Duplicate of #72?

**Comment by gdubicki** [comment]
This is fixed in [1.9.2](https://github.com/geerlingguy/ansible-role-composer/releases/tag/) and can be closed.
